libraries/AP_NavEKF/AP_Nav_Common.h

```diff
@@ -77,3 +77,5 @@ struct ekf_timing {
     float delVelDT_min;
 };
 void Log_EKF_Timing(const char *name, const uint8_t core, uint64_t time_us, const struct ekf_timing &timing);
+
+#define N_MODELS_EKFGSF 5
```

Comment:
I'm suprised 5 models work so well!
also, declare as 5U so it is unsigned, which avoids some of the casts

Reply:
Fixed